### PR TITLE
search and filter borrow layout

### PR DESCRIPTION
**Pull request by ghost**
## Description
search and filter borrow layout
## Reference Requirements
- Product Backlog ID: PBI-4 As a manager I can search/filter borrower by Name of user, Name of book.
- Task ID: 4.1 Search form layout

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:
- No

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)
No

### CODE REVIEW

**Review by at-phutran2 — COMMENTED** [review]
LGTM

File: resources/views/backend/books/borrow.blade.php [hunk 1]
```diff
@@ -14,61 +14,92 @@
...
+                                    <td>{{ date(config('define.date_format'), strtotime($borrowing->to_date)) }}</td>
+                                    <td>{{ $borrowing->date_send_mail }}</td>
+                                    <td>
+                                        <button type="button" data-action= "{{route('sendMail', $borrowing->id)}}" 
+                                        class="btn btn-warning btn-check fa-trash-o ion ion-android-drafts" data-name="{{ $borrowing->users->name }}" data-toggle="modal" data-target="#confirmSendMail" id="{{$borrowing->id}}" {{ canSendMail($borrowing->date_send_email) ? '':'disabled'}}>
+                                        </button>
+                                    </td>         
+                                </tr>
+                            @endforeach
+                            <form method="POST" action="" id="form-confirm">
```
Comment (at-xinhnguyen) [inline resources/views/backend/books/borrow.blade.php]: form này để làm gì vậy

File: resources/views/backend/books/borrow.blade.php [hunk 2]
```diff
@@ -14,61 +14,92 @@
...
+                                <tr>
+                                    <td>{{ $borrowing->users->employee_code }}</td>
+                                    <td>{{ $borrowing->users->name }}</td>
+                                    <td>{{ $borrowing->users->email }}</td>
+                                    <td>{{ $borrowing->books->name }}</td>
+                                    <td>{{ date(config('define.date_format'), strtotime($borrowing->from_date)) }}</td>
+                                    <td>{{ date(config('define.date_format'), strtotime($borrowing->to_date)) }}</td>
+                                    <td>{{ $borrowing->date_send_mail }}</td>
+                                    <td>
+                                        <button type="button" data-action= "{{route('sendMail', $borrowing->id)}}"
```
Comment (at-xinhnguyen) [inline resources/views/backend/books/borrow.blade.php]: check convention ở {{route('sendMail', $borrowing->id)}}

File: resources/views/backend/books/borrow.blade.php [hunk 3]
```diff
@@ -14,61 +14,92 @@
...
+                                    <td>{{ $borrowing->users->employee_code }}</td>
+                                    <td>{{ $borrowing->users->name }}</td>
+                                    <td>{{ $borrowing->users->email }}</td>
+                                    <td>{{ $borrowing->books->name }}</td>
+                                    <td>{{ date(config('define.date_format'), strtotime($borrowing->from_date)) }}</td>
+                                    <td>{{ date(config('define.date_format'), strtotime($borrowing->to_date)) }}</td>
+                                    <td>{{ $borrowing->date_send_mail }}</td>
+                                    <td>
+                                        <button type="button" data-action= "{{route('sendMail', $borrowing->id)}}" 
+                                        class="btn btn-warning btn-check fa-trash-o ion ion-android-drafts" data-name="{{ $borrowing->users->name }}" data-toggle="modal" data-target="#confirmSendMail" id="{{$borrowing->id}}" {{ canSendMail($borrowing->date_send_email) ? '':'disabled'}}>
```
Comment (at-xinhnguyen) [inline resources/views/backend/books/borrow.blade.php]: check convention ở chỗ này

File: resources/views/backend/books/borrow.blade.php [hunk 4]
```diff
@@ -14,61 +14,92 @@
...
+        <div class="row">
+            <div class="col-md-12">
+                <div class="box">
+                    <div class="box-header">
+                        <!-- add form search and select box for borrow -->
+                        <!-- start -->
+                        <form action="{{ route('borrowings.index') }}" method="GET" id="frm-search-borrow">
+                            <div class="form-row">
+                                <div class="form-group col-md-6 col-md-push-1">
+                                    <input type="text" class="form-control" id="" name="search-borrow" placeholder="{{ __('borrow.find_borrow') }}" value="">
```
Comment (at-xinhnguyen) [inline resources/views/backend/books/borrow.blade.php]: please remove `id` and `value`.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**